azerothcore/azerothcore-wotlk
# fix(DB/creature): Fix Arei on Ancient Spirit quest

## Changes Proposed:
-  Fixed Ariel so he walk and assist the player

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7091
- Closes https://github.com/chromiecraft/chromiecraft/issues/1270

## SOURCE:


## Tests Performed:


- .additem 11668
- .quest complete 939
- .go c id 9116
- Pick Felbound ancients
- .quest complete 4441
- Pick putified AND complete it
- .go c id 9598
- Then .go o 48849 its near the ruins.
- Use the flaute on the stop near the column (very picky)
- Check if he attacks now


## How to Test the Changes:


- .additem 11668
- .quest complete 939
- .go c id 9116
- Pick Felbound ancients
- .quest complete 4441
- Pick putified AND complete it
- .go c id 9598
- Then .go o 48849 its near the ruins.
- Use the flaute on the stop near the column (very picky)
- Check if he attacks now

Long video quest (2X speed)

https://streamable.com/bgwrbj

## Known Issues and TODO List:


- Looks like after completing the quest he should lose the ghostly body and become a real tree, but he kept being a ghost.
- With a smartAI action to change displayID should be enough i guess.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
